input/templates/my-account-forgot-password.hbs

```diff
@@ -10,49 +10,48 @@
 {{> common/header}}
 <div class="my-account">
     <div class="container">
+      {{> common/messages}}
         <div class="my-account-title">
             <span class="my-account-title-text icon-user">{{i18n "my-account:myAccount.title"}}</span>
         </div>
         <div class="row">
-            <div class="col-sm-9">
```

Comment:
Removed this div, as it was reducing the margin of the page too much and wasn't visually appealing.